$MirOS: ports/devel/pango/patches/patch-pango-uninstalled_pc_in,v 1.1 2005/09/22 19:54:57 bsiegert Exp $
--- pango-uninstalled.pc.in.orig	2006-04-29 08:43:10.000000000 +0200
+++ pango-uninstalled.pc.in	2006-12-29 02:09:48.000000000 +0100
@@ -2,6 +2,6 @@ Name: Pango Uninstalled
 Description: Internationalized text handling, Not Installed
 Version: @VERSION@
 Requires: glib-2.0,gobject-2.0,gmodule-no-export-2.0
-Libs: ${pc_top_builddir}/${pcfiledir}/pango/libpango-@PANGO_API_VERSION@.la
+Libs: @PKGCONFIG_X_LIBS@ ${pc_top_builddir}/${pcfiledir}/pango/libpango-@PANGO_API_VERSION@.la
 Cflags: -I${pc_top_builddir}/${pcfiledir}/@srcdir@
 
